dist/mixins/carousel/ds6/carousel-mixins.less

```diff
@@ -20,12 +21,17 @@
     .icon--chevron-right,
     .icon--chevron-right-small {
         margin-left: 2px;
```

Comment:
Is this needed since flexbox should center the icon?

Reply:
This is one of those things that needs to stay. Otherwise, even though the icon is centered, there is an optical illusion making it _feel_ off-center.

Reply:
I noticed that too, but I was mainly wondering if we go by 
A) What syntax is "correct" or 
B) What looks better. 
So it seems like B is the correct answer